ecommerce/managing_products/variants.rst

```diff
@@ -1,92 +1,110 @@
-==============================
-How to manage product variants
-==============================
+=======================
+Manage product variants
+=======================
 
 Product variants are used to offer variations of the same product to your
-customers on the products page. For example, the customer chooses a T-shirt
-and then selects its size and color.  In the example below, the customer
-chooses a phone, and then selects the memory; color and Wi-Fi band from the
+customers on the product's page. For example, the customer chooses a T-shirt
+and then selects its size and color. In the example below, the customer
+chooses a phone, and then selects the memory, color, and Wi-Fi band from the
 available options.
 
 .. image:: ./media/variants.png
```

Comment:
Your images must be in an individual folder.

Reply:
I am aware that for the new documentation the images must be in an individual folder, but when I first started this PR, that hadn't been established yet. However, for all new PRs that I create, I'll be sure to do that. Not a problem.